scripts/ssl-manager.js

```diff
@@ -1294,17 +1422,14 @@ function SSLManager(config) {
 
         me.getEntryPointGroup = function () {
             var group,
-                nodes,
-                resp;
-
-            resp = me.getEnvInfo();
-            if (resp.result != 0) return resp;
-
-            nodes = resp.nodes;
+                nodes;
 
+            nodes = me.getNodes();
             for (var i = 0, node; node = nodes[i]; i++) {
-                if (nodeManager.isBalancerLayer(node.nodeGroup)) {
-                    group = node.nodeGroup;
+                if (nodeManager.isBalancerLayer(node.nodeGroup) && !isBLNodeExists && node.ismaster) {
+                    nodeManager.setBalancerMasterNode(node);
+                    group = config.webroot ? config.nodeGroup : node.nodeGroup;
+                    isBLNodeExists = true;
```

Comment:
avoid global variables

Reply:
done. Created getBLNodeExists and setBLNodeExists methods